src/sentry/static/sentry/app/views/groupDetails/shared/header.jsx

```diff
@@ -188,14 +198,14 @@ const GroupHeader = createReactClass({
               )}
               <div className="count align-right m-l-1">
                 <h6 className="nav-header">{t('Events')}</h6>
-                <Link to={`${baseUrl}${groupId}/events/`}>
+                <Link to={`${baseUrl}${groupId}/events/${this.searchTermWithoutQuery()}`}>
                   <Count className="count" value={group.count} />
                 </Link>
               </div>
               <div className="count align-right m-l-1">
                 <h6 className="nav-header">{t('Users')}</h6>
                 {userCount !== 0 ? (
-                  <Link to={`${baseUrl}${groupId}/tags/user/`}>
+                  <Link to={`${baseUrl}${groupId}/tags/user/${location.search}`}>
```

Comment:
@lynnagara I've added `location.search` here for the # of distinct affected users.